src/libs/TaskUtils.ts

```diff
@@ -42,11 +38,11 @@ function getTaskReportActionMessage(action: OnyxEntry<ReportAction>): Pick<Messa
 }
 
 function getTaskTitle(taskReportID: string, fallbackTitle = ''): string {
-    const taskReport = allReports[taskReportID] ?? {};
+    const taskReport = allReports?.[taskReportID] ?? {};
```

Comment:
This should've been updated to `const taskReport = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${taskReportID}`] ?? {};` after updating how we key the reports. it caused a regression https://github.com/Expensify/App/issues/40121 More information: https://github.com/Expensify/App/issues/40121#issuecomment-2050945636